styles/chat.less

```diff
@@ -53,6 +53,22 @@
   button.confirm {
     margin-top: 0.5rem;
   }
+
+  // Award currency summary
+  .award-chat {
```

Comment:
It feels to me like we should be getting these styles for free. Is there a class or selector we can share with other contexts in which we present currency?